Resources/Prototypes/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -199,6 +199,9 @@
         Slash: 0.9
         Piercing: 0.9
         Heat: 0.9
+  - type: FlashImmunity # WD EDIT
+  - type: FlashSoundSuppression # WD EDIT
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)